addons/wowl/static/src/components/navbar/navbar.ts

```diff
@@ -1,21 +1,19 @@
-import { Component, useState } from "@odoo/owl";
+import { Component } from "@odoo/owl";
 import { useService } from "../../core/hooks";
 import { OdooEnv } from "../../types";
+import { Dropdown } from "../dropdown/dropdown";
+import { DropdownItem } from "../dropdown/dropdown_item";
 
 export class NavBar extends Component<{}, OdooEnv> {
   static template = "wowl.NavBar";
+  static components = { Dropdown, DropdownItem }
   actionManager = useService("action_manager");
   menuRepo = useService("menus");
-  state = useState({ showDropdownMenu: false });
 
   systrayItems = this.env.registries.systray.getAll();
 
-  toggleDropdownMenu() {
-    this.state.showDropdownMenu = !this.state.showDropdownMenu;
-  }
-
-  _onMenuClicked(menu: any) {
-    this.actionManager.doAction(menu.actionID, { clearBreadcrumbs: true });
-    this.state.showDropdownMenu = false;
+  _onMenuClicked(ev: any) {
```

Comment:
We should ask if handlers should start with _ or not.

Reply:
with `private _`